### ISSUE TITLE
Usage of bidi characters caused mangled display

### ISSUE THREAD

**Issue by mintty** [jesseduffield/lazygit#2774]
According to mintty/wsltty#336, lazygit seems to use some Hebrew characters (like שׂ ,שּׁ) for layout or as markers.
Since right-to-left script characters cause display to be reordered in a proper bidi way on terminals that support it,
this totally spoils any pane formatting. Please do not use bidi.

**Comment by stefanhaller** [comment]
If I understand this right, then this is a problem with Nerd Fonts version 2 that is fixed with version 3. (See https://github.com/ryanoasis/nerd-fonts/issues/478.)

Lazygit now [supports version 3](https://github.com/jesseduffield/lazygit/pull/2731) (only on master, not in the latest release, sorry), so if you could try if this fixes the issue, this would be appreciated.

**Comment by mintty** [comment]
No, it's not a font issue. The bidi behaviour is triggered by any right-to-left character, regardless of its glyph.

**Comment by mintty** [comment]
Or does the change mean that RTL characters are not used anymore?

**Comment by stefanhaller** [comment]
Yes, I know. NerdFonts is a system where existing fonts are extended with many icons. The issue was that NerdFonts v2 chose a range of code points for some of these icons that overlapped with some existing hebrew ranges. In v3 those code points were moved to an used range.

**Comment by theherk** [comment]
For anybody that wanders into this issue after unexpectedly seeing these shin characters in lieu of the expected icons, I post my scenario for posterity. At some point, the cask for nerd-fonts updated Victor Mono from v2 to v3. Probably it happened at the same time as the others, but I would only notice with Victor Mono. As a result, my my lazygit started to look a bit wonky. The solution that @stefanhaller implemented resolves this, but in case it isn't immediately clear how, you must add `nerdFontsVersion: "3"` to your configuration's `gui` section. The default is still v2, but this will get you fixed up.
